app/experimenter/jetstream/client.py

```diff
@@ -137,7 +138,11 @@ def get_other_metrics_names_and_map(data, RESULTS_METRICS_MAP):
 def get_experiment_data(experiment):
     recipe_slug = experiment.slug.replace("-", "_")
     windows = ["daily", "weekly", "overall"]
-    raw_data = {}
+    raw_data = {
+        "daily": {},
+        "weekly": {},
+        "overall": {},
```

Comment:
Strings like this, and references to them everywhere, should generally be moved to constants.  These would probably fit nicely in an enum.